.doc_gen/metadata/medical-imaging_metadata.yaml

```
@@ -301,6 +399,17 @@ medical-imaging_UpdateImageSetMetadata:
             - description:
               snippet_tags:
                 - python.example_code.medical-imaging.UpdateImageSetMetadata
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/medical-imaging
+          excerpts:
+            - description:
+              snippet_tags:
+                - medical-imaging.JavaScript.datastore.updateImageSetMetadataV3
+            - description: Demonstration of how to encode the metadata.
```

I assume this working is similar across the board. But, historically, Liz has preferred the imperative.
```suggestion
            - description: Encode the metadata.
```